src/specBuilder/combo/comboSpecBuilder.ts

```diff
@@ -20,14 +20,17 @@ import { Spec } from 'vega';
 import { BarElement, ChartChildElement, ColorScheme, ComboChildElement, ComboProps, LineElement } from '../../types';
 
 export const addCombo = produce<Spec, [ComboProps & { colorScheme?: ColorScheme; index?: number }]>(
-	(spec, { children = [], colorScheme = DEFAULT_COLOR_SCHEME, index = 0, name }) => {
+	(
+		spec,
+		{ children = [], colorScheme = DEFAULT_COLOR_SCHEME, index = 0, name, dimension = DEFAULT_TIME_DIMENSION }
+	) => {
 		const buildOrder = new Map();
 		buildOrder.set(Bar, 0);
 		buildOrder.set(Line, 0);
 
 		let { barCount, lineCount } = initializeComponentCounts();
 		const sanitizedChildren = sanitizeRscChartChildren(children);
-		const comboName = toCamelCase(name || `combo${index}`);
+		const comboName = toCamelCase(name ?? `combo${index}`);
```

Comment:
```suggestion
		const comboName = toCamelCase(name || `combo${index}`);
```

Reply:
I actually prefer this as an OR (`||`) here. If the name is an empty string, we want that to be false and use the default name instead.
Nullish coellesence (spelling?) will treat an empty string as true and use the empty string as the name.

I know this fails the code smells but I just mark them as accepted in sonarCloud. I'll see if I can figure out how to ignore this check in sonarCloud.